code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -50,13 +50,24 @@
 /datum/chemical_reaction/plasmasolidification
 	name = "Solid Plasma"
 	id = "solidplasma"
-	required_reagents = list("iron" = 5, "frostoil" = 5, "plasma" = 20)
+	required_reagents = list("frostoil" = 5, "plasma" = 20)
```

Comment:
@lordpidey exactly. It's easy. If they wanted to, they can do the same with 5 or 10 units of iron as well. So there's no real change in the balance of the recipe by removing iron. 

But the whole point is moot ANYWAYS because this reaction WONT OCCUR inside of a mob.